lib/vmfloaty/service.rb

```diff
@@ -0,0 +1,129 @@
+require 'commander/user_interaction'
```

Comment:
Awesome job on the refactor here! I had really been meaning to do something like this a long time ago, so I'm glad to see all of this done. 👍 💯